srcpkgs/nyxt/template

```diff
@@ -1,8 +1,8 @@
 # Template file for 'nyxt'
 pkgname=nyxt
-version=2.0.0
-revision=2
-wrksrc=nyxt
+version=2.2.2
+revision=1
+wrksrc=.
```

Comment:
=> nyxt-2.2.2_1: cleaning build directory...
    rm: refusing to remove '.' or '..' directory: skipping '/builddir/.'

Set create_wrksrc instead.

Reply:
So instead of having `wrksrc=.` or `wrksrc=nyxt` only have `create_wrksrc=nyxt`?